### ISSUE TITLE
Max Production numbers on getExtractors

### ISSUE THREAD

**Issue by featheredtoast** [porisius/FicsitRemoteMonitoring#27]
As promised, I did wait an entire year before finding more bugs :)

Max production numbers seem to be doubly multiplied by overclocking amount on getExtractors

EG this extractor is a fully overclocked mk3 miner on a normal node. (Even if it was pure, max prod of 1500 is still over the 1200 cap):
```
"building":"Miner Mk.3",
      "location":{
         "x":-44922.0,"y":302273.59375,"z":-2515.800049,
         "rotation":210
      },
      "Recipe":"Iron Ore",
      "production":[
         {
            "Name":"Iron Ore",
            "CurrentProd":341.200012,
            "MaxProd":1500.0,
            "ProdPercent":"22.700001"
         }
      ],
      "ManuSpeed":2.5,
      "IsConfigured":true,
      "IsProducing":false,
      "IsPaused":false,
      "CircuitID":83
```

**Comment by porisius** [comment]
I'm tempted to make some debug commands that report back to chat window what the outputs of each item are prior to being calculated. Ought to only need the first in the array of FGBuildables, but might help put this problem to bed. This isn't the first time I have heard about it. getProdStats was notorious back in the day.

**Comment by porisius** [comment]
This might help...
![image](https://user-images.githubusercontent.com/47161774/210396831-0701bf88-0ffa-4acc-9b4c-b5e88ece8e04.png)

**Comment by porisius** [comment]
Found something...
![image](https://user-images.githubusercontent.com/47161774/210412173-58b83d17-973f-42cc-8a13-c263c8f461ae.png)

This one is going to come in real handy...
![image](https://user-images.githubusercontent.com/47161774/210412402-536bf00a-b4c7-49ea-8fb5-8f548f36afae.png)

**Comment by featheredtoast** [comment]
Regarding liquids and gasses oddities, that may be related to the different units represented in game, I've opened another ticket about that stuff here: https://github.com/porisius/FicsitRemoteMonitoring/issues/28

**Comment by porisius** [comment]
It is possible... figured I would focus down one then the other... if it is a two birds/one stone scenario, high-five for the good aim?

**Comment by featheredtoast** [comment]
Fair point - I only mention it in case that was conflating confusing numbers in liquid/gas extractors.

**Comment by porisius** [comment]
Yeah, that wasn't the only spot... apparently the getProdStats was affected. FRM 0.8.15 sent to SMR.

**Comment by porisius** [comment]
![image](https://user-images.githubusercontent.com/47161774/210433275-70fe42e8-8bf5-42a4-a70c-efcd202c5378.png)
Let me know if that gets both #27 and #28

**Comment by featheredtoast** [comment]
Amazing - thanks for the quick fixes again - will be taking a look soon!

**Comment by featheredtoast** [comment]
The new numbers look much better for both issues!

One correction: `"Purity": Normal`: should be quoted in JSON `"Purity": "Normal"` for it to be parse-able

**Comment by featheredtoast** [comment]
One point of clarification, is `MaxProd` supposed to be max production the currently set `ManuSpeed` or at full 100% speed?

I'm seeing maxProd of 960 from a miner even though it's set up for 162.5% speed (780) rather than 200% (960):

```
{
      "building":"Miner Mk.3",
      "location":{
         "x":-66076.5,"y":296870.71875,"z":-5900.700195,
         "rotation":310
      },
      "Recipe":"Coal",
      "production":[
         {
            "Name":"Coal",
            "CurrentProd":636.200012,
            "MaxProd":960.0,
            "ProdPercent":"81.599998"
         }
      ],
      "Purity": Pure,
      "ManuSpeed":162.5,
      "IsConfigured":true,
      "IsProducing":true,
      "IsPaused":false,
      "CircuitID":83
}
```

**Comment by porisius** [comment]
No, and I should have thought about that... It is the Maximum Producible based on the amount of slugs inserted.

Could add a SlugPotential or just change it. Thoughts?

**Comment by featheredtoast** [comment]
I'd propose just changing it - I personally would prefer it as it makes much more sense to track current output vs max output at the set speed to catch inefficiencies/backups in production chains

**Comment by porisius** [comment]
![image](https://user-images.githubusercontent.com/47161774/210466400-dfab7c65-bee2-4d5a-b502-e02d543462da.png)

**Comment by porisius** [comment]
FRM 0.8.16 sent to SMR

**Comment by featheredtoast** [comment]
Looks like that did it - looks great!
